### PR TITLE
Add submodule: nlohmann json schema validator

### PR DESCRIPTION
**Pull request by itikhono**
### Details:
 - *Add nlohmann_json_schema_validator (https://github.com/pboettch/json-schema-validator) as submodule to validate json config files in benchmark_app and FrontEnds

### Tickets:
 - *68214*
 - *72746*

### CODE REVIEW

File: thirdparty/CMakeLists.txt [hunk 1]
```diff
@@ -192,8 +192,13 @@ endif()
 # nlohmann json
 #
 if(ENABLE_SAMPLES OR NGRAPH_PDPD_FRONTEND_ENABLE OR NGRAPH_ONNX_FRONTEND_ENABLE OR NGRAPH_TF_FRONTEND_ENABLE)
-    add_subdirectory(nlohmann_json EXCLUDE_FROM_ALL)
+    set(JSON_Install ON)
+    add_subdirectory(json/nlohmann_json EXCLUDE_FROM_ALL)
     openvino_developer_export_targets(COMPONENT openvino_common TARGETS nlohmann_json)
+
+    set(BUILD_SHARED_LIBS OFF)
+    add_subdirectory(json/nlohmann_json_schema_validator EXCLUDE_FROM_ALL)
```
Comment (ilya-lavrenov) [inline thirdparty/CMakeLists.txt:200]: it's better to wrap these 2 lines with `function` like here https://github.com/openvinotoolkit/openvino/blob/master/thirdparty/CMakeLists.txt#L35-L38

It adds new scope and we can set `BUILD_SHARED_LIBS` to `OFF` inside the function, while outside of the function variable will have default value

Comment (itikhono) [reply thirdparty/CMakeLists.txt]: fixed

**Review by ilya-lavrenov — APPROVED** [review]

**Review by ilyachur — APPROVED** [review]

**Review by ttroilov — APPROVED** [review]

File: samples/cpp/benchmark_app/CMakeLists.txt [hunk 2]
```diff
@@ -50,10 +50,10 @@ if(OpenCL_FOUND AND OpenCL_HEADERS)
 endif()
 
 # Optional OpenCV dependnency
-
 find_package(OpenCV COMPONENTS core QUIET)
 if(NOT OpenCV_FOUND)
     message(WARNING "OpenCV is disabled or not found, ${TARGET_NAME} will be built without OpenCV support. Set OpenCV_DIR")
+    target_link_libraries(${TARGET_NAME} PRIVATE nlohmann_json_schema_validator)
```
Comment (ilya-lavrenov) [inline samples/cpp/benchmark_app/CMakeLists.txt:56]: I think we also need to add lines like :
https://github.com/openvinotoolkit/openvino/blob/8453b6395a447e897c7f3a4a7a0deeb38447a606/samples/cpp/CMakeLists.txt#L118-L120
to `openvino/samples/cpp/CMakeLists.txt`

Comment (itikhono) [reply samples/cpp/benchmark_app/CMakeLists.txt:56]: thanks!
done

**Review by ilya-lavrenov — APPROVED** [review]

**Comment by itikhono** [comment]
replaced with https://github.com/openvinotoolkit/openvino/pull/9145

**Pull request closed without merging**